thrust/CMakeLists.txt

```diff
@@ -59,6 +59,10 @@ option(THRUST_ENABLE_TESTING "Build Thrust testing suite." "ON")
 option(THRUST_ENABLE_EXAMPLES "Build Thrust examples." "ON")
 option(THRUST_ENABLE_BENCHMARKS "Build Thrust runtime benchmarks." "${CCCL_ENABLE_BENCHMARKS}")
 
+# Allow the user to optionally select offset type dispatch to fixed 32 or 64 bit types
+set(THRUST_DISPATCH_TYPE "Dynamic" CACHE STRING "Select Thrust offset type dispatch." FORCE)
```

Comment:
This option is not visible to users consuming our CMake packages (e.g. CPM, `find_package(CCCL)`, `add_subdirectory(...)`, etc). They're only visible in the developer build.

Reply:
We need this option to be usable when consuming CCCL from libcudf. That was the purpose of introducing this, in #1958.